CHANGES.md

```diff
@@ -5,6 +5,27 @@ for a full listing of issues for each release.
 
 --------------------------------------------------------------------------------
 
+## 2.0a1 (2019-09-19)
+
+Rose release 59. This maintenance release is expected to be used with:
```

Comment:
Should replace this with info that this is a pre-release and is not suitable for normal use.